src/model/CoilCoolingWaterToAirHeatPumpEquationFit.hpp

```diff
@@ -114,6 +117,12 @@ namespace model {
 
     bool isRatioofInitialMoistureEvaporationRateandSteadyStateLatentCapacityDefaulted() const;
 
+    double maximumCyclingRate() const;
+
+    double latentCapacityTimeConstant() const;
+
+    double fanDelayTime() const;
```

Comment:
Cooling coil: the moved values were made required-field, setting to IDD default in the Ctor.